multinet/multinet.py

```diff
@@ -11,13 +11,24 @@
 bp = Blueprint("multinet", __name__)
 
 
+def require_db():
+    """Check if the db is live."""
+    if db.check_db():
+        return
+    else:
+        return Response(None, "500 Database Not Live")
+
+
+bp.before_request(require_db)
+
+
 def graphql_query(query, variables=None):
     """Perform a GraphQL query using optional variable definitions."""
     data = None
     errors = []
 
     result = graphql(schema, query, variables=variables or {})
-    if result:
+    if result.data:
```

Comment:
What was the reason for this change?

Reply:
This was from one of my previous revision. I rolled it back to the original code (this is the same as the current code in master)